### ISSUE TITLE
Add map slices to Vet Center locations page

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#34427]
## Issue description
The Vet Center location page includes the name, address, phone number and link to directions for the main Vet Center and any associated outstations and community access points (if any) but not the map slice. 
Map slices are generated based on lat/long provided by VAST. 

### Current experience
Map slices are shown only for the main Vet Center and only on the main page.

<details>
<summary> Screenshots </summary>

Main page display for main Vet Center
![image](https://user-images.githubusercontent.com/55411834/146981967-f35100dd-8fa0-4bf6-826e-1d0c058022be.png)

---
Locations page 

![screencapture-va-gov-nashville-vet-center-locations-2021-12-21-11_43_56](https://user-images.githubusercontent.com/55411834/146982017-8433e3d1-e4a3-42b3-a3f0-7743920f16f3.png)

</details>

### Desired experience
The Locations page includes a map slice for all listed physical locations (except Mobile Vet Centers). 

### Out of scope 
- Location displays for associated mobile Vet Centers

## Tasks
- Add map slice for listed Vet Center locations to Vet Center locations page template. 

## Acceptance criteria
The map slice is displayed on the Locations page, consistent with the main page style for 
- [ ] the main Vet Center
- [ ] any associated outstations
- [ ] any community access points
- [ ] any Vet Center facilities listed under "Other nearby Vet Centers"

**Comment by mmiddaugh** [comment]
Blocked by requirements to determine lat/long for CAPs

**Comment by mmiddaugh** [comment]
Map slice is expected for Outstations - they are locations in VAST and should have lat/long
Examples: 

- [Parkersburg Outstation with Huntington Vet Center](https://www.va.gov/huntington-vet-center/locations/)
- [Clarksville Outstation with Nashville Vet Center](https://www.va.gov/nashville-vet-center/locations/)

**Comment by GcioNapoleon** [comment]
Ticket was larger than originally expected. We were able to verify that everything is working as expected 02/18/2022

![Screen Shot 2022-02-11 at 11 36 20 AM](https://user-images.githubusercontent.com/95312667/154727264-cc413cf5-9066-4a6a-b3b0-1d72d34d8859.png)
